modules/ROOT/pages/_partials/data-sync/server-realtime-updates-amq.adoc

```diff
@@ -161,10 +196,8 @@ const client = mqtt.connect({
 const pubsub = new MQTTPubSub({ client })
 ----
 
-=== Connecting using TLS
-
-When connecting via TLS, all messages between your application and the AMQ Online broker are encrypted.
-
+.. To encrypt all messages between your application and the AMQ Online broker, use the TLS broker, for example:
```

Comment:
`use the TLS broker` doesn't really make sense. make just `use TLS` or `enable TLS`